support-frontend/assets/helpers/redux/checkout/address/validation.test.ts

```diff
@@ -264,50 +262,6 @@ describe('isValidPostcodeForHomeDelivery', () => {
 	});
 });
 
-describe('isHomeDeliveryInM25 ', () => {
-	it('returns true when the order is a home delivery and the postcode is within the M25', () => {
-		const fulfilmentOption = 'HomeDelivery';
-		const postcode = 'SE23 2AB';
-		const homeDeliveryPostcodes = ['SE23'];
-
-		const result = isHomeDeliveryInM25(
-			fulfilmentOption,
-			postcode,
-			homeDeliveryPostcodes,
-		);
-
-		expect(result).toBeTruthy();
-	});
-
-	it('returns false when the order is a home delivery and the postcode is outside the M25', () => {
-		const fulfilmentOption = 'HomeDelivery';
-		const postcode = 'DA11 7NP';
-		const homeDeliveryPostcodes = ['SE23'];
-
-		const result = isHomeDeliveryInM25(
-			fulfilmentOption,
-			postcode,
-			homeDeliveryPostcodes,
-		);
-
-		expect(result).toBeFalsy();
-	});
-
-	it('returns true when the fulfilment option is not home delivery', () => {
-		const fulfilmentOption = 'Collection';
-		const postcode = 'DA11 7NP';
-		const homeDeliveryPostcodes = ['SE23'];
-
-		const result = isHomeDeliveryInM25(
-			fulfilmentOption,
-			postcode,
-			homeDeliveryPostcodes,
-		);
-
-		expect(result).toBeTruthy();
-	});
-});
-
 describe('isHomeDeliveryAvailable', () => {
 	it('returns true when the order is a home delivery and the postcode is outside the M25 and a delivery agent is available', () => {
```

Comment:
Is this description correct? It doesn't seem like  the postcode should have to be outside the M25 for home delivery to be available.

Reply:
it doesn't have to be, have added a new test case to make this clearer